### DISCUSSION TITLE
[FAIL] Kubuntu 21.10

### DISCUSSION THREAD

**Discussion by juanpc2018** [prometheus/prometheus#10986] [General]
### What did you do?

install from synaptic

### What did you expect to see?

something

### What did you see instead? Under which circumstances?

$ prometheus
level=info ts=2022-07-02T14:20:41.494Z caller=main.go:323 msg="No time or size retention was set so using the default time retention" duration=15d
level=info ts=2022-07-02T14:20:41.494Z caller=main.go:355 msg="Starting Prometheus" version="(version=2.24.1+ds, branch=debian/sid, revision=2.24.1+ds-1ubuntu1)"
level=info ts=2022-07-02T14:20:41.494Z caller=main.go:360 build_context="(go=go1.16.5, user=pkg-go-maintainers@lists.alioth.debian.org, date=20210816-18:52:50)"
level=info ts=2022-07-02T14:20:41.494Z caller=main.go:361 host_details="(Linux 5.13.0-52-generic #59-Ubuntu SMP Wed Jun 15 20:17:13 UTC 2022 x86_64 P-7805u (none))"
level=info ts=2022-07-02T14:20:41.494Z caller=main.go:362 fd_limits="(soft=1024, hard=1048576)"
level=info ts=2022-07-02T14:20:41.494Z caller=main.go:363 vm_limits="(soft=unlimited, hard=unlimited)"
level=error ts=2022-07-02T14:20:41.495Z caller=query_logger.go:87 component=activeQueryTracker msg="Error opening query log file" file=/var/lib/prometheus/metrics2/queries.active err="open /var/lib/prometheus/metrics2/queries.active: permission denied"
panic: Unable to create mmap-ed active query log

goroutine 1 [running]:
github.com/prometheus/prometheus/promql.NewActiveQueryTracker(0x1d530e0, 0x1d, 0x14, 0x1fd1de0, 0xc0006508d0, 0x1fd1de0)
        github.com/prometheus/prometheus/promql/query_logger.go:117 +0x4e8
main.main()
        github.com/prometheus/prometheus/cmd/prometheus/main.go:391 +0x540d

$ sudo prometheus
[sudo] password: 
level=info ts=2022-07-02T14:21:29.408Z caller=main.go:323 msg="No time or size retention was set so using the default time retention" duration=15d
level=info ts=2022-07-02T14:21:29.408Z caller=main.go:355 msg="Starting Prometheus" version="(version=2.24.1+ds, branch=debian/sid, revision=2.24.1+ds-1ubuntu1)"
level=info ts=2022-07-02T14:21:29.408Z caller=main.go:360 build_context="(go=go1.16.5, user=pkg-go-maintainers@lists.alioth.debian.org, date=20210816-18:52:50)"
level=info ts=2022-07-02T14:21:29.408Z caller=main.go:361 host_details="(Linux 5.13.0-52-generic #59-Ubuntu SMP Wed Jun 15 20:17:13 UTC 2022 x86_64 P-7805u (none))"
level=info ts=2022-07-02T14:21:29.408Z caller=main.go:362 fd_limits="(soft=1024, hard=1048576)"
level=info ts=2022-07-02T14:21:29.408Z caller=main.go:363 vm_limits="(soft=unlimited, hard=unlimited)"
level=info ts=2022-07-02T14:21:29.410Z caller=web.go:468 component=web msg="Start listening for connections" address=0.0.0.0:9090
level=error ts=2022-07-02T14:21:29.410Z caller=main.go:551 msg="Unable to start web listener" err="listen tcp 0.0.0.0:9090: bind: address already in use"

**Comment by roidelapluie** [comment]
You might have a Prometheus server already running, or Cockpit launched. Something is already listening to port 9090.
